### DISCUSSION TITLE
请教一下关于设置静态路由

### DISCUSSION THREAD

**Discussion by andlink01** [kkkgo/PaoPaoGateWay#144] [问和答] [answered]
爱快 paopaodns paopaogw得组合，我看教程需要针对某些应用之类配置静态路由，但是作为一个小白，不知道哪些应用需要通过IP直连。如果没有针对这些IP配置静态路由的话会有什么影响？

**Comment by andlink01** [comment]
哪位大佬可以解答一下

**Comment by kkkgo** [comment]
影响就是用不了，比如不加tg的相关静态路由就用不了tg。

**Reply by andlink01** [reply]
> 影响就是用不了，比如不加tg的相关静态路由就用不了tg。

这个需要添加静态路由的IP有没有哪个项目定时维护更新的？有没有作业可以抄

**Reply by kkkgo** [reply]
不用抄，几乎都是不会变的IP。

**Reply by kkkgo** [reply]
既然你不清楚有什么影响，说明你也用不上。

**Reply by andlink01** [reply]
> 既然你不清楚有什么影响，说明你也用不上。

主要担心的是，不清楚哪些东西需要配置静态路由。有些东西是用别人的程序在跑。最后发现不了问题

**Reply by kkkgo** [reply, marked as answer]
常用的除了tg和网飞可能找不到其他例子了。如果你发现不了问题，那你先发现问题再问。
